public/scripts/worktrollyUI.js

```diff
@@ -66,7 +66,7 @@ $("#Marketing").click(function() {
 });
 
 $("#backlogButton").click(function() {
-    selectedStatus = "Completed";
+    backlogSprint = "-1"
```

Comment:
No need for new variable i guess, we can modify the current variables.

Reply:
ok